README.rst

```diff
@@ -6,126 +6,122 @@ Taisei
 
-    git submodule init
-    git submodule update
+Installation
+------------
 
-This step needs to be done just once, and can be skipped if you specified the
-``--recursive`` or ``--recurse-submodules`` option when cloning.
+You can find precompiled binaries of the complete game on the
+`Releases <https://github.com/taisei-project/taisei/releases>`__ page on
+GitHub, available for Windows (x86/x64), Linux, and macOS.
 
-**Important:** You should also run ``git submodule update`` whenever you pull in
-new code, checkout another branch, etc. The ``pull`` and ``checkout`` helper
-scripts can do that for you automatically.
+An experimental build for Nintendo Switch (homebrew) also exists (use at your
+own risk).
 
-Compiling from source
-^^^^^^^^^^^^^^^^^^^^^
+You can also play our experimental WebGL build through your web browser
+`here <https://play.taisei-project.org/>`__. (Chromium-based browsers and
+Firefox supported.)
 
-To build and install Taisei on \*nix, just follow these steps:
+Source Code & Development
+-------------------------
 
-::
+**Important:** As of v1.3, you cannot download GitHub's automatically-generated
+source archives, as they don't contain the necessary submodules for compiling
+the game. Make sure you either download the release tarball, or use `git clone`.
 
-    cd /path/to/taisei/source
-    mkdir build
-    cd build
-    meson --prefix=$yourprefix ..
-    ninja
-    ninja install
+You can obtain the source code of the stable releases from the
+`Releases <https://github.com/taisei-project/taisei/releases>`__ page on
+GitHub, alongside the binaries.
 
-This will install game data to ``$prefix/share/taisei/`` and build this
-path *statically* into the executable. This might be a package
-maintainer’s choice. Alternatively you may want to add
-``-Dinstall_relative=true`` to get a relative structure like
+You can also grab the latest code from git using the following commands:
 
 ::
 
-    $prefix/taisei
-    $prefix/data/
+    git clone --recurse-submodules https://github.com/taisei-project/taisei
 
-``install_relative`` is always set when building for Windows.
+The ``--recursive-submodules`` option is important as the submodules are
+necessary to compile the project.
 
-The OpenGL ES 3.0 backend is not built by default. To enable it, do:
+If you want to pull them in manually, be sure to run the following:
 
 ::
 
-    meson configure -Dr_gles30=true -Dshader_transpiler=true
+    cd taisei
+    git submodule init
+    git submodule update
+
+You should also run ``git submodule update`` whenever you pull in
+new code, checkout another branch, etc. The ``pull`` and ``checkout`` helper
+scripts can do that for you automatically.
 
-See `here <doc/ENVIRON.rst>`__ for information on how to activate it.
-Alternatively, do this to make GLES 3.0 the default backend:
+For more in-depth development and compiling instructions, see our
+`Development Guide <doc/DEVELOPMENT.rst>`__.
```

Comment:
I think it's very important to have a super basic TL;DR version of how to build and run the thing, i.e. literally
```sh
meson setup build
meson compile -C build
meson install -C build
```
Asking a potential new user/contributor to read a book that doesn't even exist yet is not the best look. On that note, I'd also remove references to not-yet-written documents, and have them added back in when they are ready.

Would be even better if all the info on how to properly obtain the source code could be further condensed somehow. Perhaps it's not necessary to mention the alternative way to init submodules… teaching people how git works is not a goal here.